.eslintrc.json

```diff
@@ -49,13 +49,6 @@
         "capitalized-comments": "off",
         "class-methods-use-this": "error",
         "comma-dangle": "off",
-        "comma-spacing": [
```

Comment:
Is there a reason why we're removing this lint rule?

Reply:
It was conflicting with itself during lint fix and prettier.  
`[x, , , , ]` vs `[x, , , ,]`